package.json

```diff
@@ -9,6 +9,10 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@auth0/auth0-vue": "^2.0.1",
+    "@vueuse/core": "^9.12.0",
+    "pinia": "^2.0.30",
```

Comment:
I'd argue we do not even need `pinia` after all for this - at least that's what I have the impression of when taking a look at the code bcc media uses (https://github.com/bcc-code/brunstadtv/blob/c111d9f5222edf03b035d91b13231b2a037b0872/web/src/services/auth.ts)

Reply:
for this we could just use the auth0, but why not start with pinia already and have the userStore where we can build some additional logic and things around auth0 as well? we do it in other apps and it works pretty nice and is handy

Reply:
btw it's still a draft PR 😄